service-csw/src/main/java/fi/nls/oskari/csw/domain/CSWIsoRecord.java

```diff
@@ -169,40 +169,53 @@ public JSONObject toJSON() {
             arr.put(onlineResource.toJSON());
         }
         JSONHelper.putValue(ret, "onlineResources", arr);
-
+        //TODO: should we create toJSON() instead of using ObjectMapper
```

Comment:
Yes, definitely. Or we could make the object model correspond to the current "JSON model" and use Jackson for serialization. Either way current implementation of mixing the two is horrible. Plus creating a new ObjectMapper is just wrong, it should be re-used.

Reply:
If you're not gonna implement this then atleast move the ObjectMapper to a static final field so that it gets atleast some reuse.